app/models/grda_warehouse/hud/client.rb

```diff
@@ -1783,11 +1783,12 @@ def last_seen_in_type(type, include_confidential_names: false)
       return nil unless type.in?(GrdaWarehouse::Hud::Project::RESIDENTIAL_PROJECT_TYPES.keys + [:homeless])
 
       service_history_enrollments.ongoing.
-        joins(:service_history_services, :project).
+        joins(:service_history_services, :project, :organization).
```

Comment:
should this be?
```suggestion
        joins(:service_history_services, project: :organization).
```
It only matters if we reference `enrollment.project.organization` vs `enrollment.organization`

Reply:
never mind, just here so we can reference `o_t[:confidential]`